src/views/DashboardPage/index.js

```diff
@@ -1 +1,74 @@
 // Dashbaord page
+import React, { Component } from "react";
+import { CognitoUserPool } from "amazon-cognito-identity-js";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import { Link, Route } from "react-router-dom";
+import HomePage from "../HomePage";
+
+const poolData = {
+  UserPoolId: process.env.REACT_APP_COGNITO_USER_POOL_ID,
+  ClientId: process.env.REACT_APP_COGNITO_CLIENT_ID,
+};
+
+const userPool = new CognitoUserPool(poolData);
+var cognitoUser = userPool.getCurrentUser();
+
+window.onload = function () {
+  if (cognitoUser != null) {
+    cognitoUser.getSession(function (err, session) {
+      if (err) {
+        alert(err);
```

Comment:
Does this mean the user isn't logged in? In which case should they be redirected to /login?